### ISSUE TITLE
waiting for apiserver: timed out waiting for the condition / pods evicted

### ISSUE THREAD

**Issue by utdrmac** [kubernetes/minikube#4468]
**The exact command to reproduce the issue**:
The initial, brand new setup, commands are as follows:
```
$ minikube config set kubernetes-version v1.13.1
$ minikube config set cpus 4
$ minikube config set memory 4096
$ minikube start
```

minkube worked fine for my work yesterday. When I came back to my laptop this morning, none of the kubectl commands would work, giving this error:
```
$ kubectl describe pod percona-xtradb-cluster-operator-6f65855d75-9cnnr
The connection to the server 192.168.99.100:8443 was refused - did you specify the right host or port?
```

I then ran `minikube stop` and then `minikube start` and this start failed.

**The full output of the command that failed**:
See 'start-stop-crash.txt'
[start-stop-crash.txt](https://github.com/kubernetes/minikube/files/3277684/start-stop-crash.txt)

**The output of the `minikube logs` command**:
I started the VM manually. SSH'd to it. Started a `journalctl -f` then ran `minikube start` in an attempt to gather all log output. See `Log at 2019-06-11 12-06-37 PM.txt`

[Log at 2019-06-11 12-06-37 PM.txt](https://github.com/kubernetes/minikube/files/3277682/Log.at.2019-06-11.12-06-37.PM.txt)

**The operating system version**:
macos 10.13.6

**Comment by medyagh** [comment]
@utdrmac  I am curious do you use a vpn ?

**Comment by utdrmac** [comment]
No VPN

**Comment by tstromberg** [comment]
This seems pretty key from your logs:

`Jun 11 16:48:51 minikube kubelet[3129]: I0611 16:48:51.906488    3129 eviction_manager.go:187] eviction manager: pods etcd-minikube_kube-system(8490cea1bf6294c73e0c454f26bdf714) evicted, waiting for pod to be cleaned up`

It sounds like your virtualbox VM has too little RAM attached to it for the pods being deployed. Please increase it to see if the problem goes away for you.

**Comment by utdrmac** [comment]
I had already doubled the amount of RAM from the default as shown above. Can probably close this as the "fix" was to simply blow everything away and start over. Same config as above, worked no issues the next time.

**Comment by tstromberg** [comment]
@author: I believe this issue is now addressed by minikube v1.4, as it should detect and present the eviction up front. If you still see this issue with minikube v1.4 or higher, please reopen this issue by commenting with `/reopen` - and be sure to include the output of `minikube logs`

Thank you for reporting this issue!
